Automattic/jetpack
# Block Theme Previews: show education modal when previewing theme

https://github.com/Automattic/jetpack/assets/1525580/4ce1aee9-9352-4416-90d0-cfc43969d865

Related to:

- https://github.com/Automattic/wp-calypso/issues/80264

## Proposed changes:

This PR shows an education modal when a user previews a block theme, for the first theme.



The modal can be dismissed and it won't be shown again, by storing a flag in a persisted redux store.

### Infra changes

I added the following in `jetpack-mu-wpcom` package:

- Support for transpiling SCSS file.
- Support for bundling CSS file into `[name]/[name].css`, similar to JS.
- New `jetpack-mu-wpcom` text domain for i18n.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






1. Patch this change by following the instructions from the bot below.
2. Go to `/themes`.
3. Click a block theme.
4. In the theme details page, click `Preview & Customize`.
5. Verify that you can see the modal as shown in the above video.
6. Verify you can dismiss the modal by clicking the primary button or the close icon.
7. Verify that the modal won't be shown anymore when previewing another theme.
8. (To re-enable the modal, you need to clear the local storage as below:)

<img width="664" alt="image" src="https://github.com/Automattic/jetpack/assets/1525580/2fcb3443-46dc-40d1-9da2-397c21b2eb55">

9. Verify that you can still see the modal when the ETK plugin is disabled.
